waspc/data/Generator/templates/react-app/src/queries/_query.js

```diff
@@ -18,4 +18,6 @@ const {= queryFnName =} = async ({ args, context }) => {
   }
 }
 
+{= queryFnName =}.useQueryKey = '{= queryRoute =}'
```

Comment:
aha ok this is sth required by 'useQuery' I suppose.

Reply:
It requires the key, and we want it to be unique per query. In our case, route is unique, so we are using that.